gap/attr.gd

```diff
@@ -89,7 +89,8 @@ DeclareAttributeThatReturnsDigraph("DigraphReverse", IsDigraph);
 DeclareAttributeThatReturnsDigraph("DigraphDual", IsDigraph);
 DeclareAttributeThatReturnsDigraph("ReducedDigraph", IsDigraph);
 DeclareAttributeThatReturnsDigraph("DigraphRemoveAllMultipleEdges", IsDigraph);
-DeclareAttributeThatReturnsDigraph("DigraphRemoveAllEdges", IsDigraph);
+DeclareAttributeThatReturnsDigraph("DigraphRemoveAllEdges", IsImmutableDigraph);
+DeclareAttributeThatReturnsDigraph("DigraphRemoveAllEdges", IsMutableDigraph);
```

Comment:
Actually just one attribute is better here, with `IsDigraph`. Then the different methods are installed for more specific filters